sonar-python-plugin/src/main/java/org/sonar/plugins/python/pylint/PylintRuleRepository.java

```diff
@@ -19,32 +19,232 @@
  */
 package org.sonar.plugins.python.pylint;
 
-import com.google.common.base.Charsets;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Map;
 import org.sonar.api.server.rule.RulesDefinition;
 import org.sonar.api.server.rule.RulesDefinitionXmlLoader;
 import org.sonar.plugins.python.Python;
-import org.sonar.squidbridge.rules.SqaleXmlLoader;
 
 public class PylintRuleRepository implements RulesDefinition {
 
   public static final String REPOSITORY_NAME = "Pylint";
   public static final String REPOSITORY_KEY = REPOSITORY_NAME;
 
   private static final String RULES_FILE = "/org/sonar/plugins/python/pylint/rules.xml";
-  private static final String SQALE_FILE = "/com/sonar/sqale/python-model.xml";
   private final RulesDefinitionXmlLoader xmlLoader;
 
+  private static final Map<String, String> REMEDIATION_COST = initializeRemediationCost();
+
   public PylintRuleRepository(RulesDefinitionXmlLoader xmlLoader) {
     this.xmlLoader = xmlLoader;
   }
 
+  private static Map<String, String> initializeRemediationCost() {
+    Map<String, String> map = new HashMap<>();
+    map.put("C0102", "1min");
+    map.put("C0103", "1min");
+    map.put("C0111", "5min");
+    map.put("C0112", "5min");
+    map.put("C0121", "20min");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "20min" 21 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1192)